data/src/main/java/ca/bc/gov/data/datasource/PatientLocalDataSource.kt

```diff
@@ -112,4 +112,10 @@ class PatientLocalDataSource @Inject constructor(
 
     suspend fun getPatientWithMedicationRecords(patientId: Long): PatientWithMedicationRecordDto? =
         patientDao.getPatientWithMedicationRecords(patientId)?.toDto()
+
+    suspend fun getPatient(patientId: Long): PatientDto = patientDao.getPatient(patientId).toDto()
+
+    suspend fun deletePatientById(patientId: Long): Int {
```

Comment:
Can we use existing code logic to delete and insert patient data. 

Ref: "patientRepository.insertAuthenticatedPatient(patient)"